templates/config.edn

```diff
@@ -189,4 +189,7 @@
 
  ;; Extra CodeMirror options
  ;; :editor/extra-codemirror-options {:keyMap "emacs" :lineWrapping true}
+
+ ;; ignore #+keyword: for parsing page references in orgmode
+ :ignored-page-references-keywords #{"author" "startup"}
```

Comment:
Better to comment this line out by default because users always have different workflows.

Reply:
Correct. Doing it.